### PR TITLE
[#6422] pagination issues fixes

### PR DESCRIPTION
**Pull request by khamui**

### CODE REVIEW

File: meinberlin/apps/budgeting/assets/PaginationButton.jsx [hunk 1]
```diff
@@ -16,6 +16,8 @@ export const PaginationButton = (props) => {
       return <i className="fa fa-chevron-left" aria-hidden="true" />
     } else if (props.type === 'next') {
       return <i className="fa fa-chevron-right" aria-hidden="true" />
+    } else if (props.type === 'mobile') {
+      return `${props.pageIndex} / ${props.pageCount}`
```
Comment (goapunk) [inline meinberlin/apps/budgeting/assets/PaginationButton.jsx:20]: the text color is grey for this and not very readable, I assume we should also use the default black here?

**Review by goapunk — CHANGES_REQUESTED** [review]
nice fixes! just one small thing

File: meinberlin/apps/budgeting/assets/Pagination.jsx [hunk 2]
```diff
@@ -31,17 +38,26 @@ export const Pagination = (props) => {
...
+            />
+          ))
+          : (
+            <PaginationButton
+              type="mobile"
+              key={getKeyStr(0, 0)}
+              pageIndex={currentPage}
+              pageCount={pageCount}
+              onClick={onPaginate}
+              isDisabled
```
Comment (goapunk) [inline meinberlin/apps/budgeting/assets/Pagination.jsx:59]: we should pass `false` here I think, otherwise the text is grey and hard to read

**Comment by khamui** [comment]
just realize #4510 is just fixed for the react pagination, still the django pagination needs to be done, but not sure if this is quite a bit of rework (at least for me).

would it be fine to do that in another PR so this can be merged already?

**Review by goapunk — APPROVED** [review]
nice, looks good to me!

**Review by phillimorland — CHANGES_REQUESTED** [review]
few questions?

File: meinberlin/assets/scss/_variables.scss [hunk 3]
```diff
@@ -55,6 +55,7 @@ $font-size-sm: 0.9rem !default;
 $font-size-xs: 0.8rem !default;
 
 // use for min-width media-queries
+$breakpoint-xxs: 23.75rem !default;
```
Comment (phillimorland) [inline meinberlin/assets/scss/_variables.scss:58]: why do you want to add an additional breakpoint? also this is added as a min width value but you use it in a max width query, we don't really want to add additional break points as it add complexity, applying the same style up to a width of 512px is pretty standard, bootstraps first breakpoint is 576 so we aren't far off the norm.

**Review by phillimorland — CHANGES_REQUESTED** [review]
cool, just a few comments and I just noticed when checking it this time that we don't use BEM, maybe we can change pagination-item to pagination__item?

File: meinberlin/assets/scss/components/_pagination.scss [hunk 4]
```diff
@@ -45,10 +48,29 @@
...
+            font-size: $font-size-xs;
+
+            &.disabled {
+                cursor: not-allowed;
+                pointer-events: none;
+                color: $border-color;
+            }
+        }
+    }
+}
```
[the remark this answers is not in the document]
Comment (phillimorland) [reply meinberlin/assets/scss/components/_pagination.scss:76]: maybe 0.7 or 0.75 just from having a quick play in the browser but your call

**Comment by fuzzylogic2000** [comment]
What is the status here? @khamui It is finished and can be reviewed again? Will you do that @phillimorland ? Or should I? (I am not sure I will notice everything you talked about.)

**Comment by phillimorland** [comment]
@fuzzylogic2000 sorry didn't see the re-tag, will take a look now

**Review by phillimorland — APPROVED** [review]
perfect! :)

**Pull request merged**